fork.yaml

```diff
@@ -26,10 +26,20 @@ def:
         - title: "Celo token duality"
           description: "The Celo token is both the native token and ERC-20 compatible."
           globs:
-            - "core/evm.go"
             - "core/vm/celo_contracts*.go"
             - "core/vm/contracts*.go"
             - "core/vm/evm.go"
+        - title: "Celo fee currencies"
+          description: "Fee currencies allow to pay transaction fees with ERC20 tokens."
+          globs:
+            - "core/evm.go"
+            - "contracts/fee_currencies.go"
+            - "core/blockchain_celo_test.go"
+            - "core/celo_evm.go"
+            - "core/state_processor.go"
+            - "core/state_transition.go"
+            - "core/types/celo_dynamic_fee_tx.go"
+            - "core/types/celo_transaction_signing.go"
```

Comment:
The following should also go here:
```
- core/txpool/*
- core/txpool/*/*
- core/types/receipt*.go
- core/types/transaction*.go
- core/types/tx*.go
```
It might catch a bit too much in the future, but since we only go with whole files, that seems to be the best we can do.